lib/logger.js

```diff
@@ -2,12 +2,43 @@
 
 const deprecate = require('depd')('propsd');
 const Winston = require('winston');
+const expressWinston = require('express-winston');
 
+/**
+ * Create a logger instance
+ * @param {string} level
+ * @param {string} filename
+ * @returns {Winston.Logger}
+ * @constructor
+ */
 function Logger(level, filename) {
-  const logLevel = level || 'info';
+  const logLevel = level.toUpperCase() || 'INFO';
+
+  /* eslint-disable rapid7/static-magic-numbers */
+  const javaLogLevels = {
```

Comment:
Java has several log utilities and "SILLY"/"VERBOSE" aren't levels I know in any. Seems like these come from Winston. Should we rename this variable to `customLogLevels` since that's what we're defining here?